src/refdb_fs.c

```diff
@@ -63,6 +66,18 @@ typedef struct refdb_fs_backend {
 	uint32_t direach_flags;
 } refdb_fs_backend;
 
+/*
+ * List of symbolic reference names that are always stored in the
+ * gitdir. This is required such that we are able to correctly
+ * resolve these references via the gitdir instead of the
+ * commondir when working trees are used.
+ */
+static const char *symrefs[] = {
```

Comment:
I think that these are often referred to as pseudorefs in git, or per-worktree refs, as they exist at the top level and exist for each worktree.